include/GafferDispatchBindings/TaskNodeBinding.inl

```diff
@@ -84,6 +84,7 @@ static boost::python::list postTasks( T &n, Gaffer::Context *context )
 template<typename T>
 static IECore::MurmurHash hash( T &n, const Gaffer::Context *context )
 {
+	IECorePython::ScopedGILRelease gilRelease;
```

Comment:
I think we need the same in `preTasks` and `postTasks` above (being careful to scope the release so that we've re-acquired before building the `python::list` :

```
GafferDispatch::TaskNode::Tasks tasks;
{
    IECorePython::ScopedGILRelease gilRelease;
    n.T::postTasks( context, tasks );
}

boost::python::list result;
for( GafferDispatch::TaskNode::Tasks::const_iterator tIt = tasks.begin(); tIt != tasks.end(); ++tIt )
{
	result.append( *tIt );
}
return result;
```